blueocean-github-pipeline/src/test/java/io/jenkins/blueocean/blueocean_github_pipeline/GithubMockBase.java

```diff
@@ -82,7 +83,7 @@ protected String CreateGithubCredential() throws UnirestException {
                 .data(ImmutableMap.of("accessToken", accessToken))
                 .status(200)
                 .jwtToken(getJwtToken(j.jenkins, user.getId(), user.getId()))
-                .put("/organizations/jenkins/scm/github/validate/")
+                .put("/organizations/" + OrganizationFactory.getInstance().list().iterator().next().getName() + "/scm/github/validate/")
```

Comment:
seems like BO could do with a method on `OrganizationFactory` to get the "default" organisation Name (not for this PR)